docs/CA-Protocol-Basics.md

```diff
@@ -34,16 +34,16 @@ to be terminated.
 ## Request - Response
 
 Each message is either a request or a response to a request that was previously sent within the same connection. Each request 
-comes with a unique message identifier and the response repeats the identifier so that the requestor can match the incoming response 
-to its corresponding request. The requestor is the one responsible for the uniqueness of request identifiers. The requestee 
+comes with a unique message identifier and the response repeats the identifier so that the requester can match the incoming response 
```

Comment:
My selection of -or was based on http://english.stackexchange.com/questions/29254/whats-the-difference-between-requester-and-requestor I would prefer to use -or instead of -er here and below